tests/tests_fctrserver.sh

```diff
@@ -167,7 +167,7 @@ fn_bootstrap_fetch_file(){
 fn_bootstrap_fetch_file_github(){
 	github_file_url_dir="${1}"
 	github_file_url_name="${2}"
-	if [ "${githubbranch}" == "master" ]&&[ "${commandname}" != "UPDATE-LGSM" ]; then
```

Comment:
Why did you modify the tests ?
Are these as well needed for this change ?

Reply:
the tests should partly match linuxgsm.sh

Reply:
i did a replace all. Would suggest having consistency throughout, to prevent random behaviour within the code